plugins/nvim-lsp/options/servers.nix

```diff
@@ -75,6 +76,10 @@ let
       languages = "text files";
       packages = [ ltex-ls ];
     };
+    lua_ls = {
+      languages = "Lua";
+      packages = [ ];
```

Comment:
Does this LSP not require any additional packages?

Reply:
The package is `lua-language-server`.

Reply:
I looked it up in [mason.nvim](https://mason-registry.dev/registry/list) There it is called lua-language-server. I think we should match that name.

Please add the package, and add a `warnIf (state <= 2211) "lua-language-server requries 23.05 or newer"`between `lua-language-server = ` and the `{`.

Thanks